server/controllers/user/internsCompleteProfile.js

```diff
@@ -30,9 +32,11 @@ module.exports = async (req, res, next) => {
 
     // update the intern profile
     if (foundProfile) {
-      await updateUserProfile(user._id, profileData);
+      const profile = await updateUserProfile(user._id, profileData);
+      pubSub.emit(pubSub.events.PROFILE_UPDATED, profile._id);
     } else {
-      await createNewProfile(profileData);
+      const profile = await createNewProfile(profileData);
+      pubSub.emit(pubSub.events.PROFILE_CREATED, profile._id);
```

Comment:
```suggestion
    const updatedProfile = await updateUserProfile(user._id, profileData);
      pubSub.emit(pubSub.events.PROFILE_UPDATED, updatedProfile);
    } else {
      const createdProfile = await createNewProfile(profileData);
      pubSub.emit(pubSub.events.PROFILE_CREATED, createdProfile);
```
My opinion here is just to give it the updated profile and no need to do a new query to the database to get the new profile :shrug: